broadinstitute/pearl
# [JN-529] more about us updates

TO TEST:
0. repopulate ourhealth
1. go to https://sandbox.ourhealth.localhost:3001/aboutUs
2. confirm Namrata's photo and bio appear
